on-the-web/index.html

```diff
@@ -12,13 +12,13 @@ <h1>Catch up with OME</h1>
         <hr class="invisible">
         <section class="social-media-feature-section">
             <div class="social-media-feature-section-outer">
-                <h3 class="social-media-feature-section-headline"><i class="fa fa-twitter" aria-hidden="true"></i> Follow Us on Twitter</h3>
+                <h3 class="social-media-feature-section-headline"><i class="fa fa-twitter" aria-hidden="true"></i> Follow Us on Bluesky</h3>
```

Comment:
Remove fa-twitter at least from this line. I do not think we can publicize bluesky link with twitter icon...